apps/web/src/app/app/(dashboard)/[workspaceSlug]/monitors/page.tsx

```diff
@@ -74,6 +74,21 @@ export default async function MonitorPage({
                   {monitor.url}
                 </dd>
               </div>
+              <div className="flex min-w-0 items-center justify-between gap-3">
+                <dt>Method</dt>
+                <dd className="overflow-hidden text-ellipsis font-semibold">
+                  <Badge
+                    className={cn(
+                      "font-mono",
+                      monitor.method === "GET"
+                        ? "bg-blue-100 text-blue-600 hover:bg-blue-100/80"
+                        : "bg-amber-100 text-amber-600 hover:bg-amber-100/80",
+                    )}
+                  >
+                    {monitor.method}
+                  </Badge>
```

Comment:
Thanks @zahid47! 🔥
Let's use the `variant="outline"` to remove the hover styles as there is no action. Instead, we can add a little border with the same color.

A suggestion:
```suggestion
                  <Badge
                    variant="outline"
                    className={cn(
                      "font-mono",
                      monitor.method === "GET"
                        ? "bg-blue-50 text-blue-600 border border-blue-100"
                        : "bg-amber-50 text-amber-600 border border-amber-100",
                    )}
                  >
                    {monitor.method}
                  </Badge>
```

Reply:
looks good! We won't need the `border` class I think because `variant="outline"` comes with one.

Reply:
Good catch on the detail! 🔥 LGTM!